src/components/Review/ReviewAdd.js

```diff
@@ -0,0 +1,94 @@
+import React, { useEffect, useState } from 'react';
+
+const ReviewAdd = () => {
+  const [menu, setMenu] = useState([
+    {
+      id: '',
+      menuName: '',
+    },
+  ]);
+
+  useEffect(() => {
+    fetch('data/menu.json')
+      .then(res => res.json())
+      .then(data => {
+        setMenu(data);
+      });
+  }, []);
+
+  return (
+    <section className="revieWrite">
+      <div className="reviewWriteWarp">
+        <h3 className="reviewWriteHead">리뷰 쓰기</h3>
+        <select name="상품선택">
+          {menu.map(menu => (
+            <option key={menu.id}>{menu.menuName}</option>
+          ))}
```

Comment:
```suggestion
          {menu.map(({id, menuName}) => (
            <option key={id}>{menuName}</option>
          ))}
```
매개변수자리에서 구조분해해주면 좀 더 깔끔하겠네요